pkg/iac/terraform/state/backend/cloud_reader.go

```diff
@@ -0,0 +1,65 @@
+package backend
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+const BackendKeyCloud = "tfcloud"
+const TerraformCloudAPI = "https://app.terraform.io/api/v2"
+
+type Attributes struct {
+	HostedStateDownloadUrl string `json:"hosted-state-download-url"`
+}
+
+type Data struct {
+	Attributes Attributes `json:"attributes"`
+}
+
+type Body struct {
```

Comment:
Since we're in a common "backend" package that is not dedicated to tfcloud, I think we should give more explicit names to types like `TFCloudBody`